src/prpy/util.py

```diff
@@ -1683,8 +1770,7 @@ def GetLinearCollisionCheckPts(robot, traj, norm_order=2, sampling_func=None):
         waypoint = traj.GetWaypoint(i)
         q1 = traj_cspec.ExtractJointValues(waypoint, robot, dof_indices)
         dq = numpy.abs(q1 - q0)
-        max_diff_float = numpy.max( numpy.abs(q1 - q0) / q_resolutions)
-        
+
```

Comment:
Pre-emptive explanation: this variable was totally unused and my auto-linter compelled me to destroy it.